PhysicsTools/PyTorch/test/testRunner.cc

```diff
@@ -1 +1 @@
-#include <Utilities/Testing/interface/CppUnit_testdriver.icpp>
+#include <Utilities/Testing/interface/CppUnit_testdriver.icpp>
```

Comment:
Missing end of line.

I see this is most files, so I assume is must be some kind of editor setting that needs to be adjusted ?